server/hello-world-lsp/webpack.lint.config.js

```diff
@@ -0,0 +1,46 @@
+var path = require('path')
+
+const baseConfig = {
```

Comment:
it seems these configs are very similar across subpacakges, would it make sense to create a root level config and share it across the subpackages

Reply:
Choice of bundler as Webpack is very specific per package, I am not ready to introduce it as a monorepo-level feature just yet. This reference configuration can be modified per subpackage based on needs